compiler/noirc_frontend/src/hir/comptime/tests.rs

```diff
@@ -31,14 +30,14 @@ fn interpret_expect_error(src: &str, func_namespace: Vec<String>) -> Interpreter
 
 #[test]
 fn interpreter_works() {
-    let program = "comptime fn main() -> pub Field { 3 }";
+    let program = "fn main() -> pub Field { 3 }";
```

Comment:
I see lots of changes from master being reverted that are unrelated to this PR here. Bad merge?